### PR TITLE
Chore: anakin and sebulba folders

### PR DESCRIPTION
**Pull request by Louay-Ben-nessir**
## What?
Changed the systems folder structure. 
## Why?
To create a folder for Sebulba systems. 
## How?
copied all of the Anakin systems into their own folder.
## Extra
This PR must be reviewed after #1080  and #1088

### CODE REVIEW

File: mava/systems/anakin/sac/ff_isac.py
Comment (sash-a) [inline mava/systems/anakin/sac/ff_isac.py:1]: The same thing needs to be done to the hydra config path in the q learning and SAC files and also the import of their types

File: mava/utils/checkpointing.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 from jax.tree_util import tree_map
 from omegaconf import DictConfig, OmegaConf
 
-from mava.systems.ppo.types import HiddenStates, Params
+from mava.systems.anakin.ppo.types import HiddenStates, Params
```
Comment (sash-a) [inline mava/utils/checkpointing.py]: We definitely shouldn't be using a PPO type here, but I guess that is not the concern of this PR.

**Review by sash-a — CHANGES_REQUESTED** [review]
Please make the changes to the sac and q learning files for the hydra config and types. Double check that it runs :pray:

**Comment by sash-a** [comment]
Just wondering if it makes sense to have the anakin/sebulba split one folder deeper so instead of `anakin/ppo` it'll be `ppo/anakin`. The reason I say this is to have a shared types files, because I assume a lot of the types we already will be useful to sebulba. @Louay-Ben-nessir have you used the types we already have like `HiddenStates`/`LearnerState` etc?

**Review by RuanJohn — APPROVED** [review]
Thank you @Louay-Ben-nessir!

**Review by sash-a — APPROVED** [review]
Thanks @Louay-Ben-nessir :fire:

**Pull request merged**